guides/source/autoloading_and_reloading_constants.md

```diff
@@ -466,9 +466,7 @@ by adding this to `config/application.rb`:
 config.autoload_paths << "#{Rails.root}/lib"
 ```
 
-`config.autoload_paths` is accessible from environment-specific configuration
-files, but any changes made to it outside `config/application.rb` don't have any
-effect.
+`config.autoload_paths` is not changeable from environment-specific configuration files.
```

Comment:
It seems to me that both sentences are saying the same thing, the first one just states that you can access it, but not change it.